<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Background" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget #Background{
background-color:rgb(102, 84, 94)}
</string>
    </property>
    <widget class="QPushButton" name="goback_button">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>720</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(102, 84, 94);
border-radius: 20px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Resource/logout.png</normaloff>../Resource/logout.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendar">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>700</width>
       <height>450</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="deadline_list">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>500</y>
       <width>700</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;; color:&quot;black&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_button">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>730</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(246, 224, 181);
font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Add Deadline</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resource/facebook.qrc"/>
 </resources>
 <connections/>
</ui>
